package-lock.json

```diff
@@ -5630,6 +5630,7 @@
         "require-from-string": "2.0.1",
         "rfc6902": "2.2.2",
         "supports-hyperlinks": "1.0.1",
+        "vm2": "github:patriksimek/vm2#468bc1e54e75e766b842830ea775669992a979e0",
```

Comment:
This file isn't related to this PR so shouldn't have changed - could you revert the changes here?